client/app/assets/less/ant.less

```diff
@@ -94,6 +94,11 @@
   }
 }
 
+// Safeguard for button anchors
```

Comment:
This does make me a little nervous -- global styles are pretty hard to lock down, and having a `button` role doesn't necessarily mean we want to make it a button.